odoo-shippable/scripts/build-image.sh

```diff
@@ -87,15 +88,23 @@ apt-get update
 apt-get upgrade
 apt-get install ${DPKG_DEPENDS} ${PIP_DPKG_BUILD_DEPENDS}
 
-# Install node dependencies
-npm install ${NPM_OPTS} ${NPM_DEPENDS}
+# Get ssl libraries before to install py37
+wget http://mirrors.edge.kernel.org/ubuntu/pool/main/o/openssl/libssl1.1_1.1.0g-2ubuntu4.1_amd64.deb -O /tmp/libssl1.1_1.1.deb \
+    && dpkg -i /tmp/libssl1.1_1.1.deb
+wget http://mirrors.edge.kernel.org/ubuntu/pool/main/o/openssl/libssl-dev_1.1.0g-2ubuntu4.1_amd64.deb -O /tmp/libssl-dev1.1.0.deb \
+    && dpkg -i /tmp/libssl-dev1.1.0.deb
+wget http://mirrors.edge.kernel.org/ubuntu/pool/main/o/openssl/openssl_1.1.0g-2ubuntu4.1_amd64.deb -O /tmp/openssl_1.1.0.deb \
+    && dpkg -i /tmp/openssl_1.1.0.deb
+install_py37
 
 # Upgrade pip for python3
-curl "https://bootstrap.pypa.io/3.2/get-pip.py" -o "get-pip.py"
-for version in '3.2' '3.3' '3.4' '3.5' '3.6'
+for version in '3.2' '3.3' '3.4' '3.5' '3.6' '3.7'
 do
     echo "Install pip for python$version"
-    python"$version" get-pip.py
+    curl "https://bootstrap.pypa.io/get-pip.py" -o "/tmp/get-pip.py"
```

Comment:
@moylop260 
Since this is inside the for loop, this same file is downloaded several times.